static/js/app.js

```diff
@@ -560,10 +590,10 @@ function makeRandColor() {
   });
 }
 
-//
-function loadJsonData(data, obliterate = false, filename = "") {
-  console.log('Displaying new data from mlalgorithm');
+// import json data
+function loadJsonData(data, filename = "", obliterate = false, ) {
```

Comment:
I don't think we use filename here, unless it's something we're working on

Reply:
The idea was to use it for alerts